flink.hcl

```diff
@@ -6,9 +6,21 @@ mutable = true
 runtime-dependencies = ["openjre-11.0.16.1_1"]
 
 version "1.15.2" {
-  source = "https://dlcdn.apache.org/flink/flink-1.15.2/flink-1.15.2-bin-scala_2.12.tgz"
+  source = "https://archive.apache.org/dist/flink/flink-1.15.2/flink-1.15.2-bin-scala_2.12.tgz"
```

Comment:
I think this could just be templatised with `${version}`.

Reply:
The scala version at the end of the URL seems to go up every so often, so if I templatised it, it would stop working after a little while when they bump the scala version.

Reply:
Aha!